rust/kcl-lib/src/execution/cad_op.rs

```diff
@@ -54,14 +54,23 @@ pub enum Operation {
         source_range: SourceRange,
     },
     GroupEnd,
+    #[serde(rename_all = "camelCase")]
+    SketchSolve {
+        /// The ID of the sketch being modified.
+        sketch_id: usize,
+        /// The node path of the operation in the source code.
+        node_path: NodePath,
+        /// The source range of the operation in the source code.
+        source_range: SourceRange,
+    },
```

Comment:
I add a new operation type, I'm not sure if this was a good idea or not, because on one hand it could be a variable declaration with `sk = sketch(on = XZ){...}` but in the case of `sketch(on = XZ){...}`, is this a special stdlibCall or not?

Either way I just needed something in the operations to move forward, happy for this to change and me to do a bit of refactoring on my end

Reply:
The new sketch block is a weird combination of stdlib function call and a code block, similar to user-defined functions. I think this is fine for now. We can change it later if it causes problems.